Formula/u/utiluti.rb

```diff
@@ -0,0 +1,42 @@
+class Utiluti < Formula
+  desc "Command-line utility to work with default apps"
+  homepage "https://github.com/scriptingosx/utiluti"
+  url "https://github.com/scriptingosx/utiluti/archive/refs/tags/v1.3.tar.gz"
+  sha256 "98d72888421cf048d157da3b64dcc4bdc165c4899de987636a1cac022e5bd7ff"
+  license "Apache-2.0"
+
+  depends_on :macos
+  depends_on macos: :big_sur
+
+  uses_from_macos "swift" => :build, since: :sonoma # swift 6.0+
+
+  patch :DATA
```

Comment:
can this patch be upstreamed?

Reply:
@chenrui333 yes, I already submitted it, however I'm not sure when the next release is expected.

Reply:
can you add a code comment to call out the upstream issue/pr?

Reply:
@chenrui333 fixed in 6a2357d.